tests/director/tests/application_templates_subscription_test.go

```diff
@@ -91,6 +97,7 @@ func TestSubscriptionApplicationTemplateFlow(baseT *testing.T) {
 		t.Run("Application is created successfully in consumer subaccount as a result of subscription", func(t *testing.T) {
 			//GIVEN
 			subscriptionToken := token.GetClientCredentialsToken(t, ctx, conf.SubscriptionConfig.TokenURL+conf.TokenPath, conf.SubscriptionConfig.ClientID, conf.SubscriptionConfig.ClientSecret, "tenantFetcherClaims")
+			expectedBaseURL := fmt.Sprintf(baseURLTemplate, DefaultSubdomain, RegionValue)
```

Comment:
Providing subdomain and region placeholders is an optional thing. The common case of stakeholders might be not to provide it.

Right now you've modified the existing test **"Application is created successfully in consumer subaccount as a result of subscription"** to work as if the two optional parameters are provided in the app template. Now we don't have such a test when they are not provided.

So it would be better to rather leave the original test as is and add a new one below it (with a clarification in the test name that it will include subdomain and region placeholders in the app template), which will test the scenario when the optional subdomain + region placeholders are part of the app template.